libdnf/module/ModulePackageContainer.cpp

```diff
@@ -357,6 +360,11 @@ bool ModulePackageContainer::isDisabled(const ModulePackage * module)
     return isDisabled(module->getName());
 }
 
+bool ModulePackageContainer::isPlatformPresent()
+{
+    return pImpl->ptatformAdded;
+}
```

Comment:
So I'm not sure I should be digging into this :smile: I haven't studied the code that much and I don't know what "platform" is here.

But it strikes me strange that the `addPlatformPackage()` function doesn't actually change the internal state of the `ModulePackageContainer` (it only calls a static function), yet you store a flag that it was called. This is weird and looks like a bad design.

But maybe I haven't looked around enough, feel free to have someone else review this, otherwise I'd need to dig more and would welcome some explanation to help me along.